packages/iTwinUI-react/package.json

```diff
@@ -0,0 +1,147 @@
+{
+  "name": "@itwin/itwinui-react",
+  "version": "1.35.0",
+  "author": "Bentley Systems",
+  "license": "MIT",
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "typings": "cjs/index.d.ts",
+  "files": ["cjs", "esm"],
+  "description": "A react implementation of our iTwinUI UX standards",
+  "homepage": "https://github.com/iTwin/iTwinUI-react",
+  "keywords": [
+    "component",
+    "components",
+    "design-system",
+    "design",
+    "framework",
+    "frontend",
+    "itwin",
+    "itwinui",
+    "react-component",
+    "react",
+    "ui",
+    "ux"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn copy-files && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
+    "build:watch": "yarn clean && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
+    "clean": "yarn clean:coverage && rimraf esm && rimraf cjs",
+    "clean:coverage": "rimraf coverage",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "createComponent": "node scripts/createComponent.js",
+    "format": "prettier --config .prettierrc **/*.{tsx,ts,js} --ignore-path .gitignore --write",
```

Comment:
Why using `.gitignore` for ignore paths? We should mix config files.

Reply:
everything that is gitignored should also be prettierignored, so it makes sense to me